Source/IntegrationTests/TestFiles/LitTests/LitTest/at-attributes/at-attributes-typos.dfy.expect

```diff
@@ -0,0 +1,8 @@
+at-attributes-typos.dfy(15,9): Error: the parameter named 'low' is already given positionally
+at-attributes-typos.dfy(13,8): Error: wrong number of arguments (got 2, but datatype constructor 'Compile' expects at most 1: (0: bool))
```

Comment:
Is it cheap to customize the error messages so that we say "attribute" instead of "datatype constructor", given that we don't yet actually define or document datatypes anywhere yet?

Alternatively, we could document the built in attributes in a prelude.dfy file just for documentation purposes, even if we don't actually parse it.

Reply:
I found how to customize the output so that it says "attribute" instead of "datatype constructor". Looks much better thanks!

Reply:
Just for the record, I think it SHOULD say "datatype constructor" when we support user-defined attributes, OR if we define a Dafny prelude that defines the built-in attributes as datatypes. But for now since we aren't, this is maximally correct.